ducktape/cluster/remoteaccount.py

```diff
@@ -39,6 +40,20 @@ def __str__(self):
         r += self.hostname
         return r
 
+    def __repr__(self):
+        return str(self.__dict__)
+
+    def __eq__(self, other):
```

Comment:
Regarding the suggestion to use`tuple(self.__dict__.items())`: are we guaranteed that the order of `self.__dict__.items()` will be consistent every time it is called?